iosHackthonApp/iosHackthonApp/Meal List Screen/MapView/MapView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  MapView.swift
+//  iosHackthonApp
+//
+//  Created by Gareth Miller on 12/11/2020.
+//  Copyright © 2020 orgName. All rights reserved.
+//
+
+import SwiftUI
+import MapKit
+
+struct MapView: UIViewRepresentable {
+
+    var annotations: [MKPointAnnotation]
+    @Binding var selectedPlace: MKPointAnnotation?
+
+    func makeUIView(context: Context) -> MKMapView {
+        let mapView = MKMapView()
+        mapView.delegate = context.coordinator
+        mapView.showsUserLocation = true
+        mapView.region = MKCoordinateRegion(center: mapView.userLocation.coordinate, latitudinalMeters: 1000, longitudinalMeters: 1000)
```

Comment:
is a `region` similar to geofence?

Reply:
I don't know, I only heard geofence for the first time when you mentioned it 😄 
I would guess so. It's specifying an area around the user to zone in on